frontend/src/reports/WhatDataAreMissing.tsx

```diff
@@ -43,6 +44,7 @@ export default function WhatDataAreMissing(props: WhatDataAreMissingProps) {
   const isPhrma = currentDropDownIds.some((condition) =>
     PHRMA_CONDITIONS.includes(condition)
   )
+  const isWisqars = currentDropDownIds.includes('gun_violence')
```

Comment:
this will only register the specific dropdown `gun_violence`, but we want it to register if youth or black men are chosen as well right? probably need something like the PHRMA check above where we check if `COMMUNITY_SAFETY_DROPDOWNIDS` includes `some()` of the currentDropDownIds. In general since we are using typescript, we should always be checking for specific entire terms, rather than using string manipulation to check for part of a string matching.